src/NBitcoin/RPC/RPCClient.cs

```diff
@@ -1236,6 +1236,17 @@ public Task SendRawTransactionAsync(byte[] bytes)
 #endregion
 
 #region Utility functions
+        /// <summary>
+        /// Returns information about a base58 or bech32 Bitcoin address
+        /// </summary>
+        /// <param name="address">a Bitcoin Address</param>
+        /// <returns>{ IsValid }</returns>
+        public ValidatedAddress ValidateAddress(BitcoinAddress address)
+        {
+            RPCResponse res = SendCommand(RPCOperations.validateaddress, address.ToString());
+            return JsonConvert.DeserializeObject<ValidatedAddress>(res.Result.ToString());
```

Comment:
And then you don't have to deserialize the result to a class like this. `res.Result` should be a boolean... So just cast it?